lib/common/CMakeLists.txt

```diff
@@ -43,7 +43,8 @@ set (HEADERS
 	${PROJECT_SOURCE_DIR}/include/vapor/Base16StringStream.h
 )
 
-add_library (common SHARED ${SRC} ${HEADERS})
+find_library(HDF5_LIB hdf5)
```

Comment:
Libraries used by the entire package are found together at the top level CMakeLists.txt

Reply:
Thanks, fixed.